internal/services/mssql/mssql_database_resource.go

```diff
@@ -775,6 +813,54 @@ func resourceMsSqlDatabaseSchema() map[string]*pluginsdk.Schema {
 				string(sql.CreateModeSecondary),
 			}, false),
 		},
+		"import": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"storage_uri": {
+						Type:     pluginsdk.TypeString,
+						Required: true,
+					},
+					"storage_key": {
+						Type:      pluginsdk.TypeString,
+						Required:  true,
+						Sensitive: true,
+					},
+					"storage_key_type": {
+						Type:     pluginsdk.TypeString,
+						Required: true,
+						ValidateFunc: validation.StringInSlice([]string{
+							string(sql.StorageKeyTypeSharedAccessKey),
+							string(sql.StorageKeyTypeStorageAccessKey),
+						}, false),
+					},
+					"administrator_login": {
+						Type:     pluginsdk.TypeString,
+						Required: true,
+					},
+					"administrator_login_password": {
```

Comment:
this should be
```suggestion
					"administrator_password": {
```

Reply:
It's the password of sql server and its schema in `azurerm_mssql_server` is `administrator_login_password`, should we make them different?
https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/mssql/mssql_server_resource.go#L78

Reply:
ahh it should be just password as login is redundant but as it already exists in schema not important enough to change, thx